LondonBritishLibrary/orient/BLorient8814.xml

```diff
@@ -0,0 +1,552 @@
+                                    </dimensions>
+                                </extent></supportDesc>
+                            <layoutDesc>
+                                <layout columns="2" writtenLines="17"/>
+                            </layoutDesc>
+                        </objectDesc> 
+                        <handDesc>
+                            <handNote script="Ethiopic" xml:id="h1">
+                                <desc>Good handwriting.</desc>
+                                <date notBefore="1700" notAfter="1799" resp="PRS8999Strelcyn"/>
+                            </handNote>
+                        </handDesc>
+                        <decoDesc>
+                            <decoNote xml:id="d1" type="miniature">
+                                <locus target="#64r"/>
+                                <desc>One or several miniatures.</desc>
+                            </decoNote>
+                            <decoNote xml:id="d2" type="miniature">
+                                <locus target="#64v"/>
+                                <desc>One or several miniatures.</desc>
+                            </decoNote>
+                            <decoNote xml:id="d3" type="miniature">
+                                <locus target="#78r"/>
+                                <desc>One or several miniatures.</desc>
+                            </decoNote>
+                            <decoNote xml:id="d4" type="miniature">
+                                <locus target="#93v"/>
+                                <desc>One or several miniatures.</desc>
+                            </decoNote>
+                        </decoDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <desc type="GuestText">Magical prayer against hail written by a second hand</desc>
+                                    <q xml:lang="gez">በስመ፡ <gap reason="ellipsis"/> አብ፡ ቍሔት፡ ወልድ፡ ቍሔት፡ መንፈስ፡ ቅዱስ፡ ቍሔት፡ <gap reason="ellipsis"/>
+                                        ኢታወርድ፡ በረደ፡ በምድረ፡ እገሌ፡ ዘእንበለ፡ ጽሩየ፡ ማየ፡ <gap reason="ellipsis"/></q>
+                                </item>
+                                <item xml:id="e1">
+                                    <locus target="#Ir"/>
+                                    <desc type="AcquisitionNote"/>
+                                    <q xml:lang="en">Bequeathed by <persName role="bequeather" ref="PRS14320DareaCurzon">Darea 
+                                        <roleName type="title">Baroness</roleName> Zouche</persName>. <date when="1917">13 Oct. 1917</date>.</q>
+                                </item>
+                                <item xml:id="e2">
+                                    <locus target="#65rb"/>
+                                    <desc type="OwnershipNote">The first owner was <persName role="owner" ref="PRS14385WaldaMadhen"/>. 
```

Comment:
I think the indication "OwnershipNote" here is somewhat misleading. Ownership note is a formulaic text mentioning the persons, written in a book a limited times (normally once). It cannot be f. 65 et passim. However, it is not quite clear how Strelcyn meant this. I assume, in the manuscript parts the names of the owners appear in supplication formulas (not ownership notes) as Strelcyn inticates. I would just list the owners under `<history/>`